examples/workshop-fomo/reach

```diff
@@ -0,0 +1,1060 @@
+#!/bin/sh
```

Comment:
Delete file from repo. Would be nice if we could show the IDE plugins how to use ../../reach instead.